share/fathead/tldr_pages/output.txt

```diff
@@ -0,0 +1,193 @@
+7za (A file archiver with high compression ratio)	A										compress directory or file\n<pre><code>7za a {{compressed.7z}} {{directory_or_file_to_compress}}</code></pre>\ndecompress an existing 7z file with original directory structure\n<pre><code>7za x {{compressed.7z}}</code></pre>\ncompress to zip format\n<pre><code>7za a -tzip {{compressed.zip}} {{directory_or_file_to_compress}}</code></pre>\ncreate multipart 7zip file; `part_size` specifies part size in Bytes, Kilobytes, Megabytes or Gigabytes.\n<pre><code>7za -v{{part_size}}{{[b|k|m|g]}} {{compressed.7z}} {{directory_or_file_to_compress}}</code></pre>	https://github.com/tldr-pages/tldr/blob/master/pages/common/7za.md
```

Comment:
Can you remove the stuff in the parens.  The first field is the search query so someone would have to search `7za (A file archiver with high compression ratio)` for this to work.